src/ontology/mondo-edit.obo

```diff
@@ -511649,6 +511649,7 @@ xref: MEDGEN:1684771 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: OMIM:618728 {source="MONDO:equivalentTo"}
 xref: UMLS:C5231478 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1684771"}
 is_a: MONDO:0003847 {source="OMIM:618728"} ! hereditary disease
+is_a: MONDO:0100510 {source="https://orcid.org/0000-0002-0587-4693"} ! spondyloepimetaphyseal dysplasia
```

Comment:
@yshwetar could you please add an additional source for this parent? (in addition to the orcid)? Thank you.